cg/meta/orders/api.py

```diff
@@ -43,18 +41,8 @@ def submit(self, project: OrderType, order_in: OrderIn, user_name: str, user_mai
         """
         submit_handler = self.submitter_registry.get_order_submitter(project)
         submit_handler.order_validation_service.validate_order(order_in)
-        # detect manual ticket assignment
-        ticket_number: str | None = self.ticket_handler.parse_ticket_number(order_in.name)
-        if not ticket_number:
-            ticket_number = self.ticket_handler.create_ticket(
-                order=order_in, user_name=user_name, user_mail=user_mail, project=project
-            )
-        else:
-            self.ticket_handler.connect_to_ticket(
```

Comment:
this method https://github.com/Clinical-Genomics/cg/blob/fc11fd53797c25ac091454b9a700f3bb2099eb5b/cg/meta/orders/ticket_handler.py#L192 will not be used anywhere, you could remove it from the ticket handler

Reply:
same for https://github.com/Clinical-Genomics/cg/blob/fc11fd53797c25ac091454b9a700f3bb2099eb5b/cg/meta/orders/ticket_handler.py#L29